package/ota_daemon/src/ota_daemon.c

```diff
@@ -33,9 +33,9 @@
 
 #include "ota_settings.h"
 
-#define OTA_RESPONSE_FILE_PATH "/data/ota/response"
-#define OTA_IMAGE_FILE_PATH "/data/ota/image"
-#define OTA_IMAGE_SHA256SUM_FILE_PATH "/data/ota/image_sha256sum"
+#define OTA_RESPONSE_FILE_PATH "/upgrade_data/ota/response"
```

Comment:
I think we can leave `ota_daemon` pointed at `/data/ota` so we don't need to bump `/upgrade_data` to 75M.  Maybe bump `/data` and `/upgrade_data` to 32m so we have some wiggle room, but if OTA always writes to `/data` and upgrade_tool always writes to `/upgrade_data` then I think we should be fine.  I think the original change was based on the faulty assumption that the upgrade_tool would decrypt into the same folder that it was upgrading from.

@pmiettinen comments?

Reply:
I'm ok with that, would there be a possible collision between manual `fileio` based downloads and `ota` in this case? Could be something to address, but at this point I'm not quite sure how.

Reply:
OTA can write to `/data/ota` and there won't be a collision for non-encrypted upgrades, for encrypted upgrades there's already a collision possible if 2 upgrades are attempting to run at the same time (because they'll both attempt to decrypt to `/upgrade_data`)... however, `upgrade_tool` prevents multiple copies of `upgrade_tool` from running at once, so I'm not sure this is possible.